### PR TITLE
ui: add route handling for UI

### PR DESCRIPTION
**Pull request by rchincha**
Signed-off-by: Ramkumar Chinchani <rchincha@cisco.com>

**What type of PR is this?**

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:

**Does this change require updates to the CNI daemonset config files to work?**:

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/api/routes.go [hunk 1]
```diff
@@ -58,14 +58,14 @@ func NewRouteHandler(c *Controller) *RouteHandler {
 }
 
 func (rh *RouteHandler) SetupRoutes() {
-	rh.c.Router.Use(AuthHandler(rh.c))
+	prefixedRouter := rh.c.Router.PathPrefix(RoutePrefix).Subrouter()
```
Comment (shimish2) [inline pkg/api/routes.go:61]: with this change I think we will be authorizing routes starting from /v2 only and /query will be skipped.

**Review by shimish2 — CHANGES_REQUESTED** [review]

File: pkg/api/routes.go (hunk 1 above)
Comment (rchincha) [reply pkg/api/routes.go:61]: yes, currently a draft PR, still evaluating implications.

Comment (rchincha) [reply pkg/api/routes.go:65]: This is fixed now. Pls take a look.

**Comment by rchincha** [comment]
Superseded by PR #670

**Pull request closed without merging**